operator/roles/forkliftcontroller/templates/api/validatingwebhookconfiguration-forklift-api.yml.j2

```diff
@@ -39,3 +39,26 @@ webhooks:
     - secrets
     scope: Namespaced
   sideEffects: None
+- admissionReviewVersions:
+  - v1
+  clientConfig:
+    service:
+      name: {{ api_service_name }}
+      namespace: {{ app_namespace }}
+      path: /static-storage-validate
+      port: 443
+  failurePolicy: Fail
+  matchPolicy: Equivalent
+  name: storages.forklift.konveyor
+  namespaceSelector: {}
+  objectSelector: {}
+  rules:
+  - apiGroups:
+    - forklift.konveyor.io
+    resources:
+    - plans
+    apiVersions:
+    - v1beta1
+    operations:
+    - '*'
```

Comment:
Do we need to validate the plan even when deleting/connecting?
I would specify the `['CREATE', 'UPDATE']`